fastlane_core/lib/fastlane_core/crash_reporter/crash_reporter.rb

```diff
@@ -24,12 +24,18 @@ def report_crash(exception: nil, action: nil)
         # a crash report), unless we have explictly turned on the crash reporter because
         # we want to test it
         return if Helper.test? && !@explitly_enabled_for_testing
-
-        payload = CrashReportGenerator.generate(exception: exception, action: action)
-        send_report(payload: payload)
-        save_file(payload: payload)
-        show_message unless did_show_message?
-        @did_report_crash = true
+        begin
+          payload = CrashReportGenerator.generate(exception: exception, action: action)
+          send_report(payload: payload)
+          save_file(payload: payload)
+          show_message unless did_show_message?
+          @did_report_crash = true
+        rescue
+          if FastlaneCore::Globals.verbose?
+            UI.error("Unable to send the crash report.")
```

Comment:
Go for it!  At least maybe we have a chance of finding out what is wrong with the crash reporter if we log the error? Downside is that it is a bit wordy and the user has to worry about two crashes at the same time (original error + crash reporter crash), which can be confusing.